Add Jetty release notes (#611)
ignitionrobotics/docs: jetty/release_notes.md

improved discovery, interoperability, and performance. To enable Zenoh, set the
environment variable `export GZ_TRANSPORT_IMPLEMENTATION=zenoh`. This allows
Gazebo to leverage Zenoh's features and potentially integrate more seamlessly
with ROS 2 and other systems utilizing Zenoh.

See the [tracking issue](https://github.com/gazebosim/gz-transport/issues/559)
and the implementation pull request
<https://github.com/gazebosim/gz-transport/pull/665> for more details.

## Remove major version from package names

We have refactored package naming to remove major version numbers, simplifying
dependency management and reducing maintenance overhead. This change impacts how
Gazebo packages are identified in CMake builds, package installations, and
import statements.

See the
[tracking issue](https://github.com/gazebo-tooling/release-tools/issues/1244)
for more details.

## Migrate from Qt5 to Qt6

With Qt5 reaching
[end of life](https://www.qt.io/blog/extended-security-maintenance-for-qt-5.15-begins-may-2025),
Gazebo has been migrated to Qt6. While there are no major changes to end-users
of the Gazebo GUI, `gz-gui` plugin authors will need to make changes to ensure

> compatibility with Qt6. Make sure to check out the

Reviewer: ```suggestion
compatibility with Qt6. Gazebo plugin maintainers should use our [migration guide](https://gazebosim.org/api/gui/10/migration_qt6.html)
to update their plugin to QT6. 
```

Reviewer: I thought this was a little awkward and could be a little stronger.

> [migration guide](https://gazebosim.org/api/gui/10/migration_qt6.html).

Reviewer: ```suggestion
```

See the [tracking issue](https://github.com/gazebosim/gz-gui/issues/586) and the
implementation pull requests <https://github.com/gazebosim/gz-gui/pull/666> and
<https://github.com/gazebosim/gz-sim/pull/2832> for more details

## Implement ROS standard simulation interfaces

Gazebo now supports the standard ROS 2
[simulation interfaces](https://github.com/ros-simulation/simulation_interfaces),
enabling seamless integration with ROS 2 ecosystems and providing access to
features for controlling and querying the simulation state. Documentation has
also been added to guide users on utilizing these new interfaces.

See the [tracking issue](https://github.com/gazebosim/ros_gz/issues/732) and the
implementation pull requests <https://github.com/gazebosim/ros_gz/pull/790> and
<https://github.com/gazebosim/docs/pull/601> for more details

## Resolve auto inertia based on input mass

> Automatically computed inertial can now use the `mass` specified in the SDF file

Reviewer: ```suggestion
Inertia parameters can now be automatically set based on an object's mass specified in an SDF file. This  
eliminates the need to manually compute and set the density of collision objects. When both mass
```
This is a little awkward and uclear. Perhaps this will help.

and no longer requires setting the density of collision objects. When both mass
and density are set, Gazebo now correctly scales the auto-computed inertia based
on the specified mass, respecting the density ratios between collisions.

See the [tracking issue](https://github.com/gazebosim/sdformat/issues/1482) and
the implementation pull request
<https://github.com/gazebosim/sdformat/pull/1513> for more details

## Improve Gazebo APIs for Reinforcement Learning

Several APIs have been improved to streamline reinforcement learning pipelines.

> We have also added an [example integration with StableBaselines3](https://github.com/gazebosim/gz-sim/tree/gz-sim10/examples/scripts/reinforcement_learning/simple_cart_pole)

Reviewer: ```suggestion
We have also added an [example integration with the StableBaselines3 Python package](https://github.com/gazebosim/gz-sim/tree/gz-sim10/examples/scripts/reinforcement_learning/simple_cart_pole)
```

portability issues of the previous approach.

See the [tracking issue](https://github.com/gazebosim/gz-tools/issues/7) and one
of the implementation pull requests
<https://github.com/gazebosim/gz-sim/pull/2849> for more details

## Parallel asset download

This enhancement significantly improves Gazebo's startup time and responsiveness
when loading worlds with many assets. Assets are now downloaded in parallel,
allowing the GUI to remain interactive during the loading process and enabling
users to close the window if needed.

See the [tracking issue](https://github.com/gazebosim/gz-sim/issues/1260) and
one of the implementation pull requests
<https://github.com/gazebosim/gz-sim/pull/2992> for more details

## Occupancy Grid Export

This enhancement adds a plugin that enables end users to directly export occupancy
grids for use with Nav2 or other mobile robotics software.

See the pull request <https://github.com/gazebosim/gz-sim/pull/2958> for more details.

## Lookup Wheel Slip system

Introduced a new system that works in conjunction with the existing `WheelSlip`
system to dynamically adjust wheel slip and friction parameters. Using an 8-bit

> slip- and friction-encoded lookup map representing the terrain the wheeled

Author: @iche033 ?
